packages/uikit/src/widgets/Menu/Menu.tsx

```diff
@@ -206,6 +208,7 @@ const Menu: React.FC<React.PropsWithChildren<NavProps>> = ({
       <AtomBox display={{ xs: "block", md: "none" }}>
         <BottomNav items={links} activeItem={activeItem} activeSubItem={activeSubItem} />
       </AtomBox>
+      {createPortal(<ScrollToTopButtonV2 />, document.body)}
```

Comment:
could you add a page filter? We might need to hide the buttongs in certain pages if needed?

Reply:
I added isShowScrollToTopButton property, is that right?